src/haproxy/haproxy.cfg

```diff
@@ -82,6 +82,7 @@ backend backend_s3
   mode http
   option forwardfor
   balance roundrobin
+  server resin_s3_1 s3:80 check port 80
```

Comment:
Technically this should be `balena_s3_1`, but given the rest of the file is littered with `resin_` I'd suggest a separate issue and we change all of them in one PR. :)

Reply:
Yep, agreed